src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -177,6 +177,14 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
     @Optional
     Long shmSize
 
+    /* 
+     * Automatically remove the container when the container's process exits.
+     * This has no effect if RestartPolicy is set.
+     */
```

Comment:
Would should indicate the version that introduces the new property with `@since`.